pandas-stubs/core/frame.pyi

```diff
@@ -2228,9 +2230,29 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def take(self, indices: list, axis: Axis = ..., **kwargs: Any) -> Self: ...
     def to_clipboard(
         self,
-        excel: _bool = ...,
-        sep: _str | None = ...,
-        **kwargs: Any,  # TODO: make more precise https://github.com/pandas-dev/pandas-stubs/issues/1174
+        excel: _bool = True,
```

Comment:
```suggestion
        excel: _bool = ...,
```
default args don't go in stubs

Reply:
Ok, I wasn't sure since I saw a few functions with parameter default literals, but most were `...`.

The [official guide for writing stubs](https://typing.python.org/en/latest/guides/writing_stubs.html#functions-and-methods) says

> For default values, use the literal values of “simple” default values (None, bools, ints, bytes, strings, and floats). Use the ellipsis literal ... in place of more complex default values. 

IMO it helps with developer experience in the IDE a bit since otherwise they'd be forced to open up the docs to see the default, but I'm happy to split the defaults off into a separate change or add a note to never write defaults into the contributing guide.